waku/v2/protocol/waku_message.nim

```diff
@@ -30,8 +30,9 @@ proc init*(T: type WakuMessage, buffer: seq[byte]): ProtoResult[T] =
   discard ? pb.getField(1, msg.payload)
   discard ? pb.getField(2, msg.contentTopic)
   discard ? pb.getField(3, msg.version)
-  discard ? pb.getField(4, msg.proof)
-  discard ? pb.getField(5, msg.timestamp)
+  discard ? pb.getField(4, msg.timestamp)
+  discard ? pb.getField(21, msg.proof)
```

Comment:
```suggestion
# XXX Experimental, this is part of https://rfc.vac.dev/spec/17/ spec and not yet part of WakuMessage spec
  discard ? pb.getField(21, msg.proof)
```

Reply:
Though this comment could possibly be part of the Nim data structure instead

Reply:
Thanks, noted this comment under the waku message type, please see the last commit